lib/widget/club_detail/member/member_list_item.dart

```diff
@@ -13,14 +13,39 @@ class MemberListItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return DataProviderWidget(
-      childBuilderTheme: (context, theme) => Container(
-        color: theme.colorsTheme.disabledWidget,
-        child: Padding(
-          padding: const EdgeInsets.all(ThemeDimens.padding12),
-          child: Container(
-            child: Text(
-              '${member.firstname!} ${member.lastname}',
-              style: theme.lightTextTheme.titleSmall,
+      childBuilderTheme: (context, theme) => Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: ThemeDimens.padding8,
+        ),
+        child: Container(
+          decoration: BoxDecoration(
+            color: theme.colorsTheme.disabledWidget,
+            borderRadius: BorderRadius.circular(ThemeDimens.largeCardBorderRadius),
+            boxShadow: [
+              BoxShadow(blurRadius: 10, color: theme.colorsTheme.shadow),
+            ],
+          ),
+          child: Padding(
+            padding: const EdgeInsets.all(ThemeDimens.padding12),
+            child: Column(
+              children: [
+                CircleAvatar(
+                  backgroundColor: theme.colorsTheme.accent,
+                  child: Text(
+                    member.firstname![0],
+                    style: theme.lightTextTheme.titleSmall.copyWith(fontWeight: FontWeight.bold),
```

Comment:
Are you sure that first name will never be null?
Also if first name is an array use .first